pyproject.toml

```diff
@@ -11,6 +11,9 @@ line_length = 88
 # are not overwritten.
 extend-exclude = "docs/sphinx-starter-pack"
 
+[tool.pyright]
+ignore = ["docs/sphinx-starter-pack"]
```

Comment:
y it's a good point. My rationale is that _it is better to gradually ignore rather than ignore the whole folder from the get-go_. We don't own the submodule so that's a no-brainer, but for the rest, I would leave it under scope for now and only ignore it in the future if it becomes an unnecessary nuisance.